docker-init-script/src/test/resources/eu/xenit/docker/alfresco/testSetCustomKeystores/ssl-keystore-passwords.properties

```diff
@@ -0,0 +1,4 @@
+aliases=ssl.alfresco.ca,ssl.repo
+keystore.password=keystore
```

Comment:
Use 3 different passwords here, to ensure the correct one gets selected

Reply:
You mean making a list property? I dont think that would be required since the documentation requires this property to be singular?

Reply:
No, I mean this and the 2 lines below should be different passwords.
Right now, you're testing if any of those passwords are selected, instead of if the correct `keystore.password` property is selected

Reply:
:ok_hand: